plasma_framework/contracts/src/vaults/Vault.sol

```diff
@@ -2,16 +2,17 @@ pragma solidity 0.5.11;
 
 import "./ZeroHashesProvider.sol";
 import "../framework/PlasmaFramework.sol";
-import "../framework/utils/Operated.sol";
+import "../utils/OnlyFromAddress.sol";
 
 /**
  * @notice Base contract for vault implementation
  * @dev This is the functionality to swap "deposit verifier".
  *      By setting new deposit verifier, we can upgrade to a new deposit tx type without upgrading the vault.
  */
-contract Vault is Operated {
+contract Vault is OnlyFromAddress {
     event SetDepositVerifierCalled(address nextDepositVerifier);
     PlasmaFramework internal framework;
+    address internal maintainer;
```

Comment:
Is it necessary to store `maintainer`? We could just call `framework.getMaintainer()` instead when it's needed